backend/src/test/java/com/bakdata/conquery/models/SerializationTests.java

```diff
@@ -150,6 +155,119 @@ public void group() throws IOException, JSONException {
 				.test(group);
 	}
 
+
+	@Test
+	public void bucketCompoundDateRange() throws JSONException, IOException {
+		Dataset dataset = new Dataset();
+		dataset.setName("datasetName");
+
+		Table table = new Table();
+
+		Column startCol = new Column();
+		startCol.setName("startCol");
+		startCol.setType(MajorTypeId.DATE);
+		startCol.setTable(table);
+
+		Column endCol = new Column();
+		endCol.setLabel("endLabel");
+		endCol.setName("endCol");
+		endCol.setType(MajorTypeId.DATE);
+		endCol.setTable(table);
+
+
+		Column compoundCol = new Column();
+		compoundCol.setName("compoundCol");
+		compoundCol.setType(MajorTypeId.DATE_RANGE);
+		compoundCol.setTable(table);
+
+		table.setColumns(new Column[]{startCol, endCol, compoundCol});
+		table.setDataset(dataset);
+		table.setName("tableName");
+
+
+		Import imp = new Import(table);
+		imp.setName("importTest");
+
+
+		DateRangeTypeCompound
+				compoundStore =
+				new DateRangeTypeCompound(startCol.getName(), endCol.getName(), new BitSetStore(BitSet.valueOf(new byte[]{8}), new BitSet()));
```

Comment:
Kannst du einen Kommentar dran machen was die 8 bedeutet, oder sie evtl sogar in binär notation schreiben? Prefix dafür ist `0b`